spec/motion-spec/matchers/include_spec.rb

```diff
@@ -0,0 +1,35 @@
+# -*- encoding : utf-8 -*-
+describe 'Matcher::Include' do
+  it 'include passes when subject include? responds true' do
+    expect([1, 2, 3, 4]).to include(4)
+  end
+
+  it 'include fails when subject include? responds false' do
+    expect_failure("[1, 2, 3, 4] expected to include \"asd\"") { expect([1, 2, 3, 4]).to include('asd') }
+  end
+
+  it 'include passes when an object responds true to include?' do
+    class TestClass; def include?(value); true;   end
+end
+    expect(TestClass.new).to include(3)
+  end
+
+  it 'include passes when an object responds false to include?' do
+    class TestClass; def include?(value); false;   end
```

Comment:
Unused method argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. You can also write as `include?(*)` if you want the method to accept any arguments but don't care about them.